client/scss/components/_listing.scss

```diff
@@ -476,8 +476,7 @@ table.listing {
   position: relative;
 
   .index {
-    color: $color-white;
-    background-color: $color-header-bg;
+    background-color: $color-white;
```

Comment:
I wonder if this could be a chance to move away from nested CSS for this page listing view, but that may well be too big a change for this MR.

Reply:
Yes, we’re going to substantially change the listing styles in an upcoming release (we already have designs), so I think we should keep the refactorings to a minimum until we do that.